README.md

```diff
@@ -0,0 +1,488 @@
+
+  * This implementation provides calls to initialize, protect, and
+    unprotect RTP packets, and makes as few as possible assumptions
+    about how these functions will be called. For example, the
+    caller is not expected to provide packets in order (though if
+    they're called more than 65k out of sequence, synchronization
+    will be lost).
+
+  * The sequence number in the rtp packet is used as the low 16 bits
+    of the sender's local packet index. Note that RTP will start its
+    sequence number in a random place, and the SRTP layer just jumps
+    forward to that number at its first invocation. An earlier
+    version of this library used initial sequence numbers that are
+    less than 32,768; this trick is no longer required as the
+    `rdbx_estimate_index(...)` function has been made smarter.
+
+  * The replay window is 128 bits in length, and is hard-coded to this
+    value for now.
+
+--------------------------------------------------------------------------------
+
+<a name="installing-and-building-libsrtp"></a>
+# Installing and Building libSRTP
+
+To install libSRTP, download the latest release of the distribution
+from [github](https://github.com/cisco/libsrtp/releases). You probably
+want to get the most recent release. Unpack the distribution and
+extract the source files; the directory into which the source files
+will go is named `libsrtp-A-B-C` where `A` is the version number, `B` is the
+major release number and `C` is the minor release number.
+
+libSRTP uses the GNU `autoconf` and `make` utilities (BSD make will not work; if
+both versions of make are on your platform, you can invoke GNU make as
+`gmake`.). In the `libsrtp` directory, run the configure script and then
+make:
+
+~~~.txt
+./configure [ options ]
+make
+~~~
+
+The configure script accepts the following options:
+
+Option                    | Description
+---------                 | -------
+\-\-help                  | provides a usage summary
+\-\-disable-debug         | compiles libSRTP without the runtime dynamic debugging system
+\-\-enable-generic-aesicm | in changes for ismacryp
+\-\-enable-syslog         | use syslog for error reporting.
+\-\-disable-stdout        | diables stdout for error reporting.
+\-\-enable-console        | use `/dev/console` for error reporting
+\-\-enable-openssl        | use OpenSSL crypto primitives
+\-\-with-openssl-dir      | Specify location of OpenSSL installation
+\-\-enable-openssl-kdf use| OpenSSL SRTP KDF algorithm
+\-\-gdoi                  | use GDOI key management (disabled at present)
+
+By default, dynamic debugging is enabled and stdout is used for
+debugging. You can use the configure options to have the debugging
+output sent to syslog or the system console. Alternatively, you can
+define `ERR_REPORTING_FILE` in `include/conf.h` to be any other
+file that can be opened by libSRTP, and debug messages will be sent to
+it.
+
+This package has been tested on the following platforms: Mac OS X
```

Comment:
Probably remove this? Not sure if it makes sense to mention specific systems when we got Travis?